src/main/java/co/inventorsoft/academy/service/TournamentService.java

```diff
@@ -0,0 +1,92 @@
+package co.inventorsoft.academy.service;
+
+import co.inventorsoft.academy.dao.MatchDAO;
+import co.inventorsoft.academy.dao.TeamDAO;
+import co.inventorsoft.academy.dao.TournamentDAO;
+import co.inventorsoft.academy.model.Match;
+import co.inventorsoft.academy.model.Team;
+import co.inventorsoft.academy.model.Tournament;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.Date;
+
+import static co.inventorsoft.academy.enums.ColorText.BLUE;
+import static co.inventorsoft.academy.enums.ColorText.CYAN;
+import static co.inventorsoft.academy.enums.ColorText.GREEN;
+import static co.inventorsoft.academy.enums.ColorText.PURPLE;
+import static co.inventorsoft.academy.enums.ColorText.RESET;
+import static co.inventorsoft.academy.enums.ColorText.YELLOW;
+
+@Component
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TournamentService {
+    /**
+     * database of teams on tournaments
+     */
+    TeamDAO teamDAO;
+    /**
+     * database of matches on tournaments
+     */
+    MatchDAO matchDAO;
+
+    /**
+     * database of tournaments
+     */
+    TournamentDAO tournamentDAO;
+
+
+    /**
+     * This method create tournament,write result to console
+     */
+    public void start() {
+        String nameOfTournament = "Derby-" + System.currentTimeMillis();
+        List<Team> teams = teamDAO.getTeams();
+        System.out.println("*****************************************************" + GREEN.getValue() + "Teams list" + RESET.getValue() + "*****************************************************" + PURPLE.getValue());
+        teams.forEach(System.out::println);
+        System.out.println(RESET.getValue() + "********************************************************************************************************************");
+        System.out.println("*****" + GREEN.getValue() + "Round" + RESET.getValue() + "************************************" + GREEN.getValue() + "Team 1" + RESET.getValue() + "***************" + GREEN.getValue() + "Team 2" + RESET.getValue() + "**********************************" + GREEN.getValue() + "Score" + RESET.getValue() + "****");
+        while (teams.size() != 1) {
+            Collections.shuffle(teams, new Random());
```

Comment:
```suggestion
            Collections.shuffle(teams);
```
Is there any special need why you supply your own `Random`?